### DISCUSSION TITLE
[Feature] Option to use the system font

### DISCUSSION THREAD

**Discussion by codenyte** [immich-app/immich#9733] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

By default Immich uses its own custom font. I think there should be an option to use the default system font instead.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by sadorowo** [comment]
AFAIK, it's a limitation of Flutter. If someday there will be a solution, I would be very happy to allow using system font by default - the app would integrate even better with my system style.

**Comment by kiyospace** [comment]
This css changes the immich Web UI font to Product Sans (copy and paste it to _Administration-->Settings-->Theme Settings-->Custom CSS_)

https://gist.github.com/Explosion-Scratch/f2986f5e76e14878ceaec5f9ac6345f2
